examples/asset_licences.md

```diff
@@ -0,0 +1,9 @@
+The "Tuffy" font is public domain, was made by Thatcher Ulrich, and was accessed from here: http://www.publicdomainfiles.com/show_file.php?id=13486218041168
+The loop doodle_pop.ogg is public domain and was made by MrZeusTheCoder.
+opengl/resources/background.png is public domain, was made by Nidhoggn, and was accessed from here: https://opengameart.org/content/backgrounds-3
```

Comment:
While it hurts to say that, but maybe it could be stored as JPG, so we don't add 600KiB of image data?

Reply:
Again, absolutely.